### PR TITLE
[stable/cluster-autoscaler] support dnsPolicy

### PR DESCRIPTION
**Pull request by so0k**
cluster-autoscaler should not rely on `kube-dns` see https://github.com/kubernetes/kops/issues/1796#issuecomment-328373547

This allows users to change the `dnsPolicy: "Default"` alleviating load on kube-dns / CoreDNS

#### Checklist

- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped to 0.13.1 
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/cluster-autoscaler/values.yaml [hunk 1]
```diff
@@ -40,9 +40,9 @@ image:
 tolerations: []
 
 extraArgs:
-  v: 4
-  stderrthreshold: info
-  logtostderr: true
+  # v: 4
```
Comment (so0k) [inline stable/cluster-autoscaler/values.yaml]: I don't see the point of adding this - it was part of https://github.com/helm/charts/pull/4614 - but this should be configurable by users, not the default chart value ...

**Comment by so0k** [comment]
Who do I need to ping to have this merged? @unguiculus / @davidkarlsen - I can fix merge conflicts, but would like to know if there's any chance this gets merged after I fix, or I should just close and keep a local fork?

**Comment by so0k** [comment]
Charts versions shouldn't just be bumped without reading the changelog of the newly released versions they're supposed to support.

**Comment by so0k** [comment]
Chart versions shouldn't just be bumped without reading the changelog of the newly released versions they're supposed to support.

**Comment by so0k** [comment]
Chart versions shouldn't just be bumped without reading the changelog of the newly released versions they're supposed to support.

**Comment by so0k** [comment]
Chart versions shouldn't just be bumped without reading the changelog of the newly released versions they're supposed to support.

**Comment by so0k** [comment]
Chart versions shouldn't just be bumped without reading the changelog of the newly released versions they're supposed to support.

File: stable/cluster-autoscaler/values.yaml (hunk 1 above)
Comment (linki) [reply stable/cluster-autoscaler/values.yaml]: Let's keep this change out of this PR. Then we can merge the `dnsPolicy` related changes.

**Comment by Multiply** [comment]
Anything we can do to get this PR in?

File: stable/cluster-autoscaler/values.yaml (hunk 1 above)
Comment (so0k) [reply stable/cluster-autoscaler/values.yaml]: @linki - Reverted - but why is the verbosity so high by default and could you please instruct me how to remove these extraArgs if they are set in the Chart defaults?

**Comment by so0k** [comment]
@Multiply - I've updated my branch but honestly I've forked the chart as it's almost impossible to get anything merged on this repo.

File: stable/cluster-autoscaler/values.yaml (hunk 1 above)
Comment (linki) [reply stable/cluster-autoscaler/values.yaml]: @so0k I don't know why they are set to these values. I'd prefer to have it similar to other charts but I would need to check a couple. We can tackle this in a seprate PR.

In the meantime, shouldn't you be able to override the values with something that logs less, e.g. in `values.yaml`
```yaml
extraArgs:
  v: 1
```
or `--values extraArgs.v=1` etc. on the command line?

Comment (so0k) [reply stable/cluster-autoscaler/values.yaml]: Ok thanks!

**Comment by so0k** [comment]
any update @paulczar

**Comment by so0k** [comment]
should I rebase again or should we just close and I use a fork as it seems most ppl use forks from helm stable as they lag behind

**Comment by so0k** [comment]
@paulczar let's close this, I'll maintain my own fork as many features lacking

**Comment by stealthybox** [comment]
This isn't really an effective place to bring this up, but I find the OWNERS file based governance of `helm/charts` to be ineffective in comparison to how we merged changes here before.

Good patches are dying all the time now.

**Comment by echoboomer** [comment]
Please consider merging this one as a lot of people could benefit from it.

**Pull request merged**